bigtable/CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Changes
 
+## v1.5.0
+- Add support for managed backups.
```

Comment:
I'm surprised this is the only change. `release-please` and Conventional Commits will be really nice when it's all working.

Reply:
Yep-- previous release was < 1 month ago, and the only other changes were dependency updates :)